flutter/lib/src/sentry_flutter.dart

```diff
@@ -180,7 +189,8 @@ mixin SentryFlutter {
     }
 
     final renderer = options.rendererWrapper.getRenderer();
-    if (!platformChecker.isWeb || renderer == FlutterRenderer.canvasKit) {
+    if (!MultiViewHelper.isMultiViewEnabled() && !platformChecker.isWeb ||
```

Comment:
same as above we could also move the check inside the integration and maybe log there